features/auto-refresh.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Auto Refresh</title>
+</head>
+<body>
+<p><a href="../index.html">[Home]</a></p>
+
+<h1>Website that refreshes every 3 seconds. You can use this to validate URL bar focus.</h1>
```

Comment:
```suggestion
<h1>Website that refreshes every 3 seconds.</h1>
<p>You can use this to validate that the URL bar keeps focus between reloads.</p>
```

I didn't first get it from the description, so I propose we expand it a bit.